### PR TITLE
Release 6.4.0

### PR DESCRIPTION
**Pull request by Schparky**
### Fixed
- Ignore any empty attributes in RestAPI attribute list.
- Dependency updates.
- Upgrade Go to 1.19.

_Also includes some refactoring to the configuration structure. I was considering other ways of providing configuration that would not store secrets in plain text within AWS services._

### Included
- #79 
- #80 
- #81 
- #82 
- #83 
- #86

### CODE REVIEW

File: google/users.go [hunk 1]
```diff
@@ -182,8 +182,8 @@ func isMatchingType(i interface{}, findType string) map[string]interface{} {
 	return nil
 }
 
-// setStringFromInterface gets a string from an interface{}, and assigns it to a map
-func setStringFromInterface(i interface{}, m map[string]string, key string) {
+// setStringFromInterface gets a string from an any, and assigns it to a map
```
Comment (Baggerone) [inline google/users.go:185]: You don't need to change this, but I wonder what a better term for "an any" would be -- "a generic"?

**Review by Baggerone — APPROVED** [review]

File: google/users.go (hunk 1 above)
Comment (jason-jackson) [reply google/users.go:185]: Or just a `value`.

**Review by jason-jackson — APPROVED** [review]

File: google/users.go (hunk 1 above)
Comment (Schparky) [reply google/users.go:185]: Huh, I don't know. I'll come up with something because it doesn't make sense at the moment.

Comment (Schparky) [reply google/users.go:185]: The technical term is "empty interface". I'll use that.

**Pull request merged**